src/network/connection.cpp

```diff
@@ -492,10 +492,10 @@ SharedBuffer<u8> IncomingSplitBuffer::insert(BufferedPacketPtr &p_ptr, bool reli
 
 void IncomingSplitBuffer::removeUnreliableTimedOuts(float dtime, float timeout)
 {
-	std::deque<u16> remove_queue;
+	std::vector<u16> remove_queue;
```

Comment:
Deuque is more efficient for insertions (push_back).

Reply:
@SmallJoker,

- `remove_queue` as `std::vector`

https://stackoverflow.com/questions/35483716/is-stddeque-faster-than-stdvector-for-inserting-at-the-end#answer-42743181